app/email.go

```diff
@@ -619,6 +631,28 @@ func (es *EmailService) SendAtUserLimitWarningEmail(email string, locale string,
 	return true, nil
 }
 
+// SendAdminOverTheLimitAlertEmail formats an email template and sends an email to an admin specified in the email arg
+func (es *EmailService) SendAdminOverTheLimitAlertEmail(tu int64, cul int, user, email, locale, siteURL string) (bool, *model.AppError) {
+	T := utils.GetUserTranslations(locale)
+
+	subject := T("api.templates.over_limit_subject", map[string]interface{}{"UserName": user})
+
+	bodyPage := es.newEmailTemplate("user_limit_admin_notification", locale)
+	bodyPage.Props["Title"] = T("api.templates.at_limit_title")
+	bodyPage.Props["Info4"] = T("api.templates.at_limit_stats", map[string]interface{}{"TotalUsers": tu, "CloudUserLimit": cul})
```

Comment:
Which email template is used for this email? It seems that the [Figma](https://www.figma.com/file/uv9A6dZsekQHHhOhYC7yc2/Cloud-UX?node-id=7861%3A888) linked in the ticket didn't have any refernece to the amount of seats taken / limit: 

<img width="730" alt="Screen Shot 2021-02-09 at 5 43 07 PM" src="https://user-images.githubusercontent.com/936315/107438336-4eba4600-6afe-11eb-8ac2-3a967289c422.png">

Reply:
Sorry. I just thought it would be useful for the admin to have some more context on why they're getting the email. But I have removed it now